src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -151,6 +151,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $tag = $this->findProperTagName($input, $errorIo, $object, $tag);
             $options = ['tag' => $tag];
         } elseif ($name = $input->getArgument('name')) {
+            if (!$input->getOption('show-arguments')) {
+                $errorIo->warning('Not passing the --show-arguments option is deprecated when a service name is provided.');
+            }
+
```

Comment:
You can revert this change since the option no longer does anything.

Reply:
The option needs to be deprecated though